example/ios/Podfile.lock

```diff
@@ -695,4 +695,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: b47e0ec631d71c9367d2a243b641bb32727ea571
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.12.1
```

Comment:
is that ok if I push a 1.15.0 version here? updated my cocoapods to latest

Reply:
I'd say yeah lets undo the Podfile.lock change since that is unrelated to the build.gradle change of note here. This would get updated on a `pod install`. Technically, it should use the version from the Gemfile https://github.com/appcues/appcues-react-native-module/blob/main/example/ios/Gemfile#L4 and use `bundle exec pod install`. Of course, it's all just the example app and nothing customer facing.